source/statements.tex

```diff
@@ -843,10 +843,10 @@
 \end{note}
 \begin{note}
 A \tcode{return} statement can involve
-an invocation of a constructor to perform a copy or move of the operand
+a copy or move of the operand
 if it is not a prvalue or if its type differs from the return type of the function.
-A copy operation associated with a \tcode{return} statement can be elided or
-converted to a move operation if an automatic storage duration variable is returned\iref{class.copy.elision}.
+The copy operation associated with a \tcode{return} statement can be elided or
+converted to a move operation if an implicitly movable entity\iref{class.copy.elision} is returned.
```

Comment:
The implicitly movable entity is no longer in class.copy.elision. The copy elision still is.